proto/src/determined/api/v1/api.proto

```diff
@@ -726,6 +728,54 @@ service Determined {
     };
   }
 
+  // Create training metrics.
+  rpc CreateTrainingMetrics(CreateTrainingMetricsRequest)
+      returns (CreateTrainingMetricsResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/training_metrics"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "Internal"
+    };
+  }
+
+  // Update training metrics.
+  rpc UpdateTrainingMetrics(UpdateTrainingMetricsRequest)
```

Comment:
I would call these `Patch`s since they only update the parts provided. Also, fwiw, with that scheme as opposed to using field masks / an actual patch, we can't differentiate update to default from unprovided. Not sure I see a huge issue with that, but it's something to think about.